### DISCUSSION TITLE
pr预览使用surge.sh部署有文件数量限制

### DISCUSSION THREAD

**Discussion by mengqiuleo** [opentiny/tiny-vue#1381] [General]
这个bug解决不掉，我的分包方式有问题， #1375

**Comment by mengqiuleo** [comment]
用 vercel preview ？

**Reply by kagol** [reply]
得看下 vercel 是否也有这个限制

**Reply by mengqiuleo** [reply]
应该可以，vercel限制是 15,000：https://vercel.com/docs/limits/overview#files

现在 vite 原生打包后的 js 文件有：3059

`@demos` 文件夹可以优化，如果不使用 vite 插件拷贝，使用自定义脚本拷贝，优化后的数量是：3733

所以最后的文件数量应该 不到 8000
